packages/view/src/ui-view.tsx

```diff
@@ -42,7 +42,9 @@ const GlobalStyle = createGlobalStyle<privateViewProps>`
         ColorTokens.token_100
       )};`}
     `}
+
     font-weight: 400;
+    width: 100%;
```

Comment:
I noticed that this style was missing in the DOM so width is tied to the 100% of the screen.